scripts/generate-regression-tests.sh

```diff
@@ -7,7 +7,18 @@ kollect() {
     local name="$1"
     shift
     echo '#!/bin/sh' > "$name.sh"
-    "$@" --save-temps --dry-run | xargs $KORE/scripts/kollect.sh "$name" >> "$name.sh"
+    "$@" | xargs $KORE/scripts/kollect.sh "$name" >> "$name.sh"
+    chmod +x "$name.sh"
+}
+
+kollect-file() {
+    local name="$1"
+    local path="$2"
+    shift
+    shift
```

Comment:
Why did you use `shift` here?
Could this two calls be replaced with one `shift 2` call?

Reply:
`kollect-file` works similarly to `kollect`, but it also takes the output file path as an argument, and we need to shift past that argument now as well. You're right that it can be replaced with `shift 2`.